### ISSUE TITLE
Feature Request: progress history?

### ISSUE THREAD

**Issue by bensaufley** [TortugaPower/BookPlayer#948]
## Feature request

I'm constantly accidentally tapping somewhere on the play bar and ending up in a completely different section of the book I'm listening to. It's happened an annoyingly large number of times. It would be great if there were some kind of way to easily get back to where I was.

## Narrative

I say "progress history" because it seems like a simple "back button" doesn't really make sense with the incremental progress of audio.

**GIVEN:**
- I have navigated to a distinct timestamp in the book
**WHEN:**
- I open the "Progress History" sheet
**THEN:** 
- I am shown my current location (assuming the book has been playing since I navigated), the starting location that I had navigated to, and the location I had navigated from (and more history?)

Basically, something that keeps track of _jumps_ in time (from/to) and allows me to navigate to where I was before the jump.

## Design

I expect this could just be a simple TableView list, with the timestamp in the book and maybe the datetime of when the jump occurred

**Comment by GianniCarlo** [comment]
Hey @bensaufley ! there is something already to kind of keep a history of the last place after an accidental jump, if you go to the Bookmarks list, you should see a 'Last time before time skip'.

Does the accidental jump happen when it's on the lock screen? if so, we have in mind to add a preference to lock that functionality, as it's way to easy to just brush the now playing widget and it jumping away 🤔 

![IMG_B58B3E0993FD-1](https://github.com/TortugaPower/BookPlayer/assets/14112819/b2c7a555-81d5-46b9-aac1-3a8d4e48b36c)

**Comment by bensaufley** [comment]
Holy smokes! That seems like exactly what I need! I had no idea!

Yes, it's usually on the lock screen. Occasionally it's on my Apple Watch, too (I also end up starting a completely different book, randomly, because of my watch—I really don't need my watch to default to BookPlayer but I can't seem to stop it from doing so)

**Comment by bensaufley** [comment]
And it actually just immediately came in handy, because the latest testFlight appears to have knocked me back to 00:00 😁

Seems like we're good here!
